src/Dialect/ONNX/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ add_onnx_mlir_library(OMONNXOps
         IndexExprDetail.cpp)
 
 add_dependencies(OMONNXOps OMONNXOpsInc)
+add_dependencies(OMONNXRewriteIncGen OMONNXOpsIncGen)
```

Comment:
I would keep the dependencies closer to where the targets are declared. In this case the add_onnx_mlir_rewriter calls. It makes the file easier to read when they are co-located.